overview.md

```diff
@@ -1,9 +1,18 @@
 ---
-title: How it Works
+title: Overview
 layout: singlePage
-sectionid: howItWorks
+sectionid: overview
 ---
 
+## What is the Language Server Protocol?
+Implementing language support like auto complete, goto definition, or type hovering for a programming language for different development tools is a significant effort. Each development tool provides different APIs for language extenders. Moreover, an extension has to be implemented in the implementation language of the tool itself.

```

Comment:
Rewording and flow suggestions. Take what parts you like:

> Implementing support for features like autocomplete, goto definition, or documentation on hover for a programming language is a significant effort. Traditionally this work must be repeated for each development tool, as each provides different APIs for implementing the same features.

> The idea behind a <i>Language Server</i> is to provide the language-specific smarts inside a server that can communicate with development tooling over inter-process communication. The idea behind the <i>Language Server Protocol</i> is to standardize how tools and servers communicate, so that a single language server can talk to many different development tools, and a single tool can talk to many language servers!